esphome/components/nextion/display.py

```diff
@@ -133,6 +137,9 @@ async def to_code(config):
     var = cg.new_Pvariable(config[CONF_ID])
     await uart.register_uart_device(var, config)
 
+    if max_queue_age := config.get(CONF_MAX_QUEUE_AGE):
+        cg.add(var.set_max_queue_age(max_queue_age))
```

Comment:
If for some reason someone puts `0` as the value in config (which is valid with the schema) then this block of code will not run as it will be falsy and either this code should be altered, or set a min range on the validators.

```suggestion
    if (max_queue_age := config.get(CONF_MAX_QUEUE_AGE, None)) is not None:
        cg.add(var.set_max_queue_age(max_queue_age))
```